build/gulpfile.extensions.js

```diff
@@ -24,6 +24,19 @@ const plumber = require('gulp-plumber');
 const ext = require('./lib/extensions');
 
 const extensionsPath = path.join(path.dirname(__dirname), 'extensions');
+// {{SQL CARBON EDIT}}
+const sqlNonBuiltInExtensions = [
```

Comment:
I think else where in the build scripts we refer to these as the built-in extensions (see https://github.com/microsoft/azuredatastudio/blob/master/build/lib/extensions.ts#L228).  It might be good to either use a different name here or there since it's confusing to use opposite terms to describe the same thing.

Reply:
Done. And also limited the listed to only the extensions that have been added to loc package.json at this time. Will test rest and add incrementally.